server/controllers/member/patchUsername.js

```diff
@@ -0,0 +1,23 @@
+const yup = require('yup');
+
+const { patchUsername } = require('../../database/queries/members/patchUsername');
+
+const patchUserNameSchema = yup.object().shape({
+  username: yup.string().required(),
+  memberId: yup.number().required(),
+});
+
+module.exports = (req, res, next) => {
+  const { memberId } = req.params;
+  const { username } = req.body;
+  patchUserNameSchema.validate({ username, memberId })
+    .then(() => {
+      patchUsername(username, memberId)
+        .then(({ rows }) => {
+          if (!rows[0]) return next({ code: 400, msg: 'Bad Request ..' });
+          return res.status(200).send({ error: null, data: rows[0].username });
```

Comment:
why did you need a `return`?

also, why did you send only the username?

![image](https://user-images.githubusercontent.com/18149438/59737132-16009580-9265-11e9-8c4c-cf5e7bf735a0.png)